app/src/main/res/layout/emoji_pref_item.xml

```diff
@@ -0,0 +1,119 @@
+<?xml version="1.0" encoding="utf-8"?>
+<android.support.constraint.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:layout_width="match_parent"
+    android:layout_height="72dp">
+
+    <!--This is a thumbnail picture-->
+    <com.pkmmte.view.CircularImageView
+        android:id="@+id/emojicompat_thumb"
+        android:layout_width="48dp"
+        android:layout_height="48dp"
+        android:layout_marginStart="16dp"
+        android:layout_marginTop="12dp"
+        android:layout_marginBottom="12dp"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent"
+        app:srcCompat="@drawable/ic_emoji_24dp"/>
+
+    <!--This is the font's name-->
+    <TextView
+        android:id="@+id/emojicompat_name"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginStart="72dp"
+        android:layout_marginTop="8dp"
+        android:text="Emoji font"
+        android:textSize="16sp"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent" />
+
+    <!--A short caption...-->
+    <TextView
+        android:id="@+id/emojicompat_subtitle"
+        android:layout_width="0dp"
+        android:layout_height="wrap_content"
+        android:layout_marginTop="8dp"
+        android:layout_marginEnd="72dp"
+        android:text="It's the best. Everyone says so!!"
+        app:layout_constraintTop_toBottomOf="@id/emojicompat_name"
+        app:layout_constraintStart_toStartOf="@id/emojicompat_name"
+        app:layout_constraintHorizontal_bias="0"
+        app:layout_constraintEnd_toEndOf="parent"/>
+
+    <!--This progress bar is shown while the font is downloading.-->
+    <ProgressBar
+        android:id="@+id/emojicompat_progress"
+        style="?android:attr/progressBarStyleHorizontal"
+        android:layout_width="0dp"
+        android:layout_height="wrap_content"
+        android:layout_marginEnd="72dp"
+        android:layout_marginTop="8dp"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="@id/emojicompat_name"
+        app:layout_constraintTop_toBottomOf="@id/emojicompat_name"
+        android:indeterminate="true"
+        android:visibility="gone"/>
+
+    <!--Click on it and the font will be downloaded!-->
+    <ImageButton
+        android:id="@+id/download_emoji_button"
+        android:layout_width="72dp"
+        android:layout_height="72dp"
+        android:paddingBottom="36dp"
+        android:paddingTop="36dp"
+        android:paddingEnd="36dp"
+        android:paddingStart="36dp"
+        android:layout_marginEnd="-20dp"
+        android:layout_marginStart="-20dp"
+        style="?android:attr/borderlessButtonStyle"
+        android:background="@drawable/button_ripple"
```

Comment:
should be `?attr/selectableItemBackgroundBorderless`, delete the `button_ripple` drawable